.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,31 @@
+name: Unittests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Is 3.8 already available here? Perhaps `3.8-dev`?

Reply:
Alas, no. 

```
Available versions:

2.7.16 (x64)
3.5.7 (x64)
3.6.9 (x64)
3.7.4 (x64)
##[error]Node run failed with exit code 1
```

Is this a reason to keep travis around? I think 3.8 is going to take a while before people will use it. I am fine with ignoring 3.8 for now, you?

Reply:
Travis doesn't have it either. It was worth trying, but no problem if we don't have it.